src/env.cc

```diff
@@ -250,6 +255,13 @@ Environment::~Environment() {
   // CleanupHandles() should have removed all of them.
   CHECK(file_handle_read_wrap_freelist_.empty());
 
+  // dispose the Persistent references to the compileFunction
+  // wrappers used in the dynamic import callback
+  for (auto & entry : compile_fn_entries) {
```

Comment:
style nit: left-leaning `&`, i.e. `auto&`, or – this is just personal preference: 

```suggestion
for (CompileFnEntry* entry : compile_fn_entries)
```

Reply:
Sure.